pages/api/flatfile/get-filefeed-actions.ts

```diff
@@ -22,9 +22,19 @@ export default async function handler(
     throw new Error("No session");
   }
 
-  const actions = await getActions(
+  const eventActions = await getActions(
     token.organizationId,
     ActionType.FileFeedEvent
   );
+
+  const syncActions = await getActions(
+    token.organizationId,
+    ActionType.SyncFilefeedRecords
+  );
```

Comment:
I would change getActions to take in a type [] and look up all of them in one query

Reply:
Edited